swarm/swarm.go

```diff
@@ -149,15 +149,16 @@ func NewSwarm(ctx *node.ServiceContext, backend chequebook.Backend, config *api.
 	db := storage.NewDBAPI(self.lstore)
 	delivery := stream.NewDelivery(to, db)
 	// TODO: decide on intervals store file location
-	intervalsStore, err := intervals.NewDBStore(filepath.Join(config.Path, "stream-intervals.db"))
+	intervalsStore, err := state.NewDBStore(filepath.Join(config.Path, "stream-intervals.db"))
 	if err != nil {
 		return
 	}
 	self.streamer = stream.NewRegistry(addr, delivery, self.lstore, intervalsStore, false)
 	stream.RegisterSwarmSyncerServer(self.streamer, db)
 	stream.RegisterSwarmSyncerClient(self.streamer, db)
 
-	self.bzz = network.NewBzz(bzzconfig, to, nil)
+	stateStore, err := state.NewDBStore(filepath.Join(config.Path, "state-store.db"))
```

Comment:
should we use one store for this and the `intervalsStore`? would make sense.. thoughts?

Reply:
yes i would prefer